packer-build/start.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -ex
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+sudo chmod 666 /dev/kvm # it fails otherwise
```

Comment:
This is not a good idea. Add the `vagrant` user to the `libvirtd` group as explained here: https://wiki.alpinelinux.org/wiki/KVM on alpine or to the `kvm` group on Debian-based distros, then remove this line from this script.

If this script is an example for the user, write a readme section on how to set up those permissions on their host.